mappings/net/minecraft/client/render/OverlayTexture.mapping

```diff
@@ -3,9 +3,12 @@ CLASS net/minecraft/class_4608 net/minecraft/client/render/OverlayTexture
 	FIELD field_21444 DEFAULT_UV I
 	METHOD method_23209 setupOverlayColor ()V
 	METHOD method_23210 getU (F)I
+		ARG 0 whiteOverlayProgress
 	METHOD method_23212 getV (Z)I
+		ARG 0 hurt
 	METHOD method_23213 teardownOverlayColor ()V
-	METHOD method_23624 packUv (FZ)I
-	METHOD method_23625 packUv (II)I
+	METHOD method_23624 getUV (FZ)I
+		ARG 1 hurt
+	METHOD method_23625 packUV (II)I
```

Comment:
```suggestion
	METHOD method_23625 packUv (II)I
```

Reply:
hmm, why did you apply the other suggestion but not this one

Reply:
Oops, applied this one too.